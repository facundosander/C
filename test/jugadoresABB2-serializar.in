crearArbolJugadores

# Construyo arbol 

# -- Primer jugador Juan 7 --
crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 7 21 Uruguay Juan
insertarJugadorArbolJugadores

# -- Segundo jugador Roberto 20 --
crearJugada
1 4 3 2 1
agregarEnPartida
crearJugador 20 19 Brasil Roberto
insertarJugadorArbolJugadores

# -- Tercer jugador Luis 2 --
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugador 2 21 Uruguay Luis
insertarJugadorArbolJugadores


# imprimo arbol
imprimirArbolJugadores

# Transformo arbol en lista
serializarArbolJugadores

liberarArbolJugadores

Fin