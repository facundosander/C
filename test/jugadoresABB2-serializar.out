1>2>#  Construyo arbol .
3>#  -- Primer jugador Juan 7 --.
4>5>6>7>8>#  -- Segundo jugador Roberto 20 --.
9>10>11>12>13>#  -- Tercer jugador Luis 2 --.
14>15>16>17>18>#  imprimo arbol.
19>Jugador 2:  Luis, Uruguay, 21 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 7:  Juan, Uruguay, 21 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
Jugador 20:  Roberto, Brasil, 19 años
Jugada: 1 
Movimientos: Abajo Arriba Izquierda Derecha 
20>#  Transformo arbol en lista.
21>Jugador 7:  Juan, Uruguay, 21 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
Jugador 2:  Luis, Uruguay, 21 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugador 20:  Roberto, Brasil, 19 años
Jugada: 1 
Movimientos: Abajo Arriba Izquierda Derecha 
22>23>Fin.
